core/templates/dev/head/components/state-editor/state-editor.directive.ts

```diff
@@ -62,13 +66,17 @@ oppia.directive('stateEditor', [
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/components/state-editor/state-editor.directive.html'),
       controller: [
-        '$scope', '$rootScope', 'INTERACTION_SPECS', 'StateEditorService',
-        'StateContentService', 'StateHintsService', 'StateSolutionService',
-        'StateInteractionIdService', 'StateCustomizationArgsService',
+        '$rootScope', '$scope', 'StateContentService',
+        'StateCustomizationArgsService',
+        'StateEditorService', 'StateHintsService',
```

Comment:
Can you please use the 80 spaces wherever it's possible?

Reply:
Done